pkg/providers/imagefamily/bootstrap/aksbootstrap.go

```diff
@@ -164,11 +163,7 @@ var (
 				Subnet: "aks-subnet", // xd
 			},
 		},
-		NetworkConfig: &nbcontractv1.NetworkConfig{
-			VnetCniPluginsUrl: vnetCNILinuxPluginsURL, // - [currently required, installCNI in provisioning scripts depends on CNI_PLUGINS_URL]
-			CniPluginsUrl:     cniPluginsURL,          // - [currently required, same]
-		},
-		IsVhd: true, // s
+		NetworkConfig: &nbcontractv1.NetworkConfig{},
```

Comment:
Can be done later, but maybe we should also initialize nested configs in the constructor as well? that way we dont have to declare empty configs like this

Reply:
Forgot to remove this. We did initialize it in constructor.
Thanks for catching this.